dev-tools/xtask/Cargo.toml

```diff
@@ -31,6 +34,10 @@ clap.workspace = true
 dev-tools-common.workspace = true
 fs-err.workspace = true
 macaddr.workspace = true
+nexus-db-model.workspace = true
+# See omicron-rpaths for more about the "pq-sys" dependency.
+pq-sys = "*"
+semver.workspace = true
```

Comment:
Note to self: The dependency addition here sucks; it's probably worth pulling this into a new binary like we did with the downloader

Reply:
Done in 7637e78c99a8bc885807a212f5905dc215a23f05